src/ios/Reporting.m

```diff
@@ -1,3 +1,4 @@
+#import <Webkit/Webkit.h>
```

Comment:
Would this work correctly on older iOS versions? I think the `WKWebView` is only available on iOS 8+, and I'm not sure if we're ready to drop iOS 7 support completely.